ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -1775,19 +1771,20 @@ def _get_dtype_inputs_for_batch_matmul(draw):
         lhs = draw(
             helpers.array_values(
                 dtype=dtype[0],
-                shape=(batch_size, dim_size, dim_size),
+                shape=(batch_size, dim_size_left, dim_size_right),
                 min_value=2,
                 max_value=5,
             )
         )
         rhs = draw(
             helpers.array_values(
                 dtype=dtype[0],
-                shape=(batch_size, dim_size, dim_size),
+                shape=(batch_size, dim_size_right, dim_size_left),
                 min_value=2,
                 max_value=5,
             )
         )
+
```

Comment:
The function actually works with shape of any size, not just 3D. 
Example:
```
import numpy as np
import jax.lax as lax

A = np.zeros((6, 2, 3, 5, 4))
B = np.zeros((6, 2, 3, 4, 5))

C = lax.batch_matmul(A, B)

print(C.shape)  # (6, 2, 3, 5, 5)
```
Basically, this function can be applied on any two matrices that have either the exact same shape or same shape with the last two dimension sizes swapped.
Instead of sampling the sizes first and passing them to `shape`, I would suggest you first sample `lhs` without specifying a `shape` and then using its randomly generated shape to create a shape for `rhs`. Hope this makes sense!